src/components/buttonFavorites/ButtonFavorites.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useContext, useState } from "react";
+import VideoContext from "../../providers/VideoContext";
+import * as Style from "../styles/styleSheet.js";
```

Comment:
Just import what you'll be using, like: 
 
`import { Button } from "../styles/styleSheet.js"; `

So when you use the Button component, you just call `<Button>` instead of `<Style.Button>`.
This way, code looks cleaner 😄